distributed/client.py

```diff
@@ -660,11 +660,11 @@ def __init__(
                 security = getattr(self.cluster, "security", None)
 
         if security is None:
-            security = Security()
+            security = TLSSecurity()
```

Comment:
It's not clear how to handle this case.  The user hasn't provided any security object explicitly.  Whlie assuming `TLSSecurity` makes sense today we might want to generalize this default approach in the future.

Reply:
how about replacing TLSSecurity() with security.get_default_security(). At the moment, it would return TLSSecurity as expected, but could be changed to return something from the configuration files.